bgpd/bgp_nb_config.c

```diff
@@ -6102,13 +6621,1229 @@ int bgp_global_afi_safis_afi_safi_ipv4_unicast_redistribution_list_rmap_policy_i
 	return NB_OK;
 }
 
+static int
+bgp_global_afi_safis_admin_distance_modify(struct nb_cb_apply_finish_args *args)
+{
+	struct bgp *bgp;
+	const struct lyd_node *af_dnode;
+	const char *af_name;
+	afi_t afi;
+	safi_t safi;
+	uint8_t distance_ebgp, distance_ibgp, distance_local;
+
+	bgp = nb_running_get_entry(args->dnode, NULL, true);
+
+	af_dnode = yang_dnode_get_parent(args->dnode, "afi-safi");
+	af_name = yang_dnode_get_string(af_dnode, "./afi-safi-name");
+	yang_afi_safi_identity2value(af_name, &afi, &safi);
+
+	distance_ebgp = yang_dnode_get_uint8(args->dnode, "./external");
+	distance_ibgp = yang_dnode_get_uint8(args->dnode, "./internal");
+	distance_local = yang_dnode_get_uint8(args->dnode, "./local");
+
+	if (bgp->distance_ebgp[afi][safi] != distance_ebgp
+	    || bgp->distance_ibgp[afi][safi] != distance_ibgp
+	    || bgp->distance_local[afi][safi] != distance_local) {
+		bgp->distance_ebgp[afi][safi] = distance_ebgp;
+		bgp->distance_ibgp[afi][safi] = distance_ibgp;
+		bgp->distance_local[afi][safi] = distance_local;
```

Comment:
Since you're doing everything in this `apply_finish()` callback, please remove the `/* TODO: implement me. */` boilerplate code from the other corresponding callbacks.